src/PrivateRoute.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react'
+import { Route, Redirect } from 'react-router-dom'
+import Menubar from './components/Menubar'
+import Navbar from './components/Navbar'
+
+export const PrivateRoute = ({ component: Component, ...rest }) => (
+    <>
+    <Navbar/>
+    <Route {...rest} render={props => (
+        localStorage.getItem('token')
+        ?
+        <>
+        <Menubar/>
+        <Component {...props} />
+        </>
```

Comment:
Can you place this inside `()`? At the first look, the use of the ternary operator is confusing.

Reply:
Not clear?? @sidvenu

Reply:
```suggestion
        (
          <>
          <Menubar/>
          <Component {...props} />
          </>
        )
```